### ISSUE TITLE
Feature Request: A command to open crossbow from the pallet

### ISSUE THREAD

**Issue by clubsandwish** [shoedler/crossbow#33]
It would be very useful to open crossbow without needing to find the icon to click on.

**Comment by shoedler** [comment]
Makes sense. I'll add this once I get back to maintaining this. In the mean time - feel free to create a pull request to add it yourself :)
